### DISCUSSION TITLE
Which platforms should i use solely for image generating using ControlNet,LORA, hires fix & etc?

### DISCUSSION THREAD

**Discussion by fishonainternet** [TheLastBen/fast-stable-diffusion#2436] [General]
Which one is more better and cost efficient? Runpod? Paperspace? or Google Colab Pro? Can anybody give me comparison or pro & cons using one of this platform solely for image generation that includes Controlnet, LORA, hiresfix & etc?

**Comment by TheLastBen** [comment]
On thing that stands out it google colab is extremely slow compared to the alternatives, and it offers only T4, (A100 is too expensive for SD)
